pkg/snmp/snmp.go

```diff
@@ -144,20 +144,40 @@ func (c *Config) BuildSNMPParams() (*gosnmp.GoSNMP, error) {
 
 	var authProtocol gosnmp.SnmpV3AuthProtocol
 	lowerAuthProtocol := strings.ToLower(c.AuthProtocol)
-	if lowerAuthProtocol == "md5" {
+	if lowerAuthProtocol == "none" {
```

Comment:
(minor) Is `lowerAuthProtocol == "none"` needed ? Doesn't **not setting** the option already mean NoAuth?

Same comment for the other case.